package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@streammedev/parrot-server",
+  "version": "1.0.0",
+  "description": "Simple http server which returns a body with information about the request such as method, pathname, query parameters, and the request body. It is useful for testing.",
+  "main": "index.js",
+  "keywords": [
+    "test"
+  ],
+  "bin": {
+    "parrot-server": "./bin/parrot-server"
+  },
+  "author": {
+    "name": "StreamMe",
+    "url": "http://github.com/streammedev"
+  },
+  "license": "UNLICENSED",
+  "repository": {
+    "type": "git",
+    "url": "ssh://git@git.vubeology.com/npm/simple-test-server.git"
+  },
+  "publishConfig": {
+    "registry": "http://npm-onsite-internal.stream.me:8080"
+  },
+  "scripts": {
+    "test": "npm run babel && happiness && mocha --compilers js:babel-register --grep 'BROWSER: ' --invert",
+    "prepublish": "npm test",
+    "postpublish": "git push $$ git push --tags",
+    "test-browser": "mochify",
+    "babel": "babel src -d lib",
+    "dev": "babel src -d lib --watch",
+    "api-docs": "rm API.md && jsdoc2md src/index.js >> API.md"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.16.0",
```

Comment:
Look through these, I am pretty sure some are not needed, or improperly installed as dev when they are deps.